### DISCUSSION TITLE
NOAA Forecast API 'wrh.noaa.gov/forecast/xml/xml.php' not working

### DISCUSSION THREAD

**Discussion by tinkerj33** [weather-gov/api#465] [Q&A]
I use NOAA API 'wrh.noaa.gov/forecast/xml/xml.php' which stopped working from early this year. Do you know which is the new working URL that replaced this one? Thanks in advance for the help.

**Comment by gerrysworkbench** [comment]
Hi there!  Also running into the same issue with the wrh.noaa.gov/forecast/xml/xml.php endpoint and was wondering if you ever found a replacement or workaround. Would really appreciate any insight. Thanks in advance!

**Reply by dwhitemv25** [reply]
You replied to a 3+ year old post. 

This is so old I can't find good descriptions of what that service actually provided. Best information I can find about it is that there is no drop-in replacement.

I think the closest replacement to it is the XML output from the `/gridpoints/.../forecast` endpoint in API. That is in a different XML schema than the old URL, though. Add a 'Accept: application/vnd.noaa.dwml+xml' header to your API request to get it. 

Not likely worth the work to use XML if you can help it though. The JSON output is a lot easier to deal with in modern tools.
